addons/website_sale_delivery/models/sale_order.py

```diff
@@ -64,9 +64,16 @@ def _check_carrier_quotation(self, force_carrier_id=None, keep_carrier=False):
         return bool(carrier)
 
     def _get_delivery_methods(self):
-        address = self.partner_shipping_id
+        def _is_carrier_available(carrier):
+            # Drop carriers where price computation fails (no price rule available/matching request)
+            res = carrier.rate_shipment(self)
+            return res['success']
```

Comment:
As you're changing that in stable, wouldn't it be safer with a `get`?
```suggestion
            return res.get('success')
```